src/desktop/apps/identity_verification/routes.tsx

```diff
@@ -0,0 +1,56 @@
+import { buildServerApp } from "reaction/Artsy/Router/server"
+import { buildServerAppContext } from "desktop/lib/buildServerAppContext"
+import { routes } from "reaction/Apps/IdentityVerification/routes"
+import { stitch } from "@artsy/stitch"
+// const metaphysics = require("lib/metaphysics.coffee")
+
+export const identityVerification = async (req, res, next) => {
+  if (!res.locals.sd.CURRENT_USER) {
+    return res.redirect(
+      `/login?redirectTo=${encodeURIComponent(req.originalUrl)}`
+    )
+  }
+  try {
+    const {
+      bodyHTML,
+      redirect,
+      status,
+      headTags,
+      scripts,
+      styleTags,
+    } = await buildServerApp({
+      routes,
+      url: req.url,
+      userAgent: req.header("User-Agent"),
+      context: buildServerAppContext(req, res),
+    })
+
+    if (redirect) {
+      res.redirect(302, redirect.url)
+      return
+    }
+
+    // Render layout
+    const layout = await stitch({
+      basePath: __dirname,
+      layout:
+        "../../components/main_layout/templates/react_minimal_header.jade",
+      blocks: {
+        head: () => headTags,
+        body: bodyHTML,
+      },
+      locals: {
+        ...res.locals,
+        assetPackage: "identity_verification",
+        headerLogoHref: res.locals.sd.APP_URL,
```

Comment:
lets double check if we need this

Reply:
I _think_ this is needed, in the sense that if we're not in the new app shell (outside of A/B test) we defer to old page layout with the artsy logo in the top right. But if in the new one, then everything is [contained inside of reaction](https://github.com/artsy/reaction/blob/master/src/Apps/Auction/Routes/ConfirmBid/index.tsx#L299-L305)

Reply:
I was really referring to `hideHeaderOnEigen`, which Erik has removed.